.github/workflows/php-cs-fixer.yml

```diff
@@ -0,0 +1,27 @@
+name: PHP CS Fixer
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    paths:
+      - '**.php'
+
+jobs:
+  php-cs-fixer:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Install
+        run: composer install
```

Comment:
Maybe `- uses: php-actions/composer@v2` could be more effective here?

Reply:
Sgtm